Makefile

```diff
@@ -0,0 +1,153 @@
+SHELL = /bin/bash
+
+.DEFAULT_GOAL := help
+
+
+CURRENT_DIR := $(shell pwd)
+
+# Test reports configuration
+TEST_REPORT_DIR ?= $(CURRENT_DIR)/tests/report
+TEST_REPORT_FILE ?= nose2-junit.xml
+
+# general targets timestamps
+TIMESTAMPS = .timestamps
+REQUIREMENTS := $(TIMESTAMPS) $(PIP_FILE) $(PIP_FILE_LOCK)
+
+# Find all python files that are not inside a hidden directory (directory starting with .)
+PYTHON_FILES := $(shell find ./* -type d \( -path ./build -o -path ./dist \) -prune -false -o -type f -name "*.py" -print)
+
+# PIPENV files
+PIP_FILE = Pipfile
+PIP_FILE_LOCK = Pipfile.lock
+
+# default configuration
+ENV_FILE ?= .env.local
+
+# Commands
+PIPENV_RUN := pipenv run
+PYTHON := $(PIPENV_RUN) python3
+PIP := $(PIPENV_RUN) pip3
+YAPF := $(PIPENV_RUN) yapf
+ISORT := $(PIPENV_RUN) isort
+NOSE := $(PIPENV_RUN) nose2
+PYLINT := $(PIPENV_RUN) pylint
+
+PACKAGE_VERSION = $(shell awk '/^Version:/ {print $$2}' gatilegrid.egg-info/PKG-INFO)
+
+
+all: help
+
+
```

Comment:
Personally I like the way `service-control` implemented the help part:
https://github.com/geoadmin/service-control/blob/develop/Makefile#L178